docs/howtos/layers/vectors.md

```diff
@@ -11,30 +11,82 @@ kernelspec:
   name: python3
 ---
 
-# Using the vectors layer
+# Using the `vectors` layer
 
-In this document, you will learn about the `napari` `Vectors` layer, including
+In this document, you will learn about the `napari` `vectors` layer, including
 how to display many vectors simultaneously and adjust their properties. You will
-also understand how to add a vectors layer and edit it from the GUI and from the
-console.
+also understand how to add a vectors layer. Vector layers can be added only
+programmatically, i.e., console, python script.
```

Comment:
I think the part about being added/created only programmatically should be more emphasized.

Reply:
👍

Reply:
Nadalyn - create an admonition or other special effect paragraph to emphasize this point.